docs/layer_util.js

```diff
@@ -195,20 +224,25 @@ function addLayerFromId(map, layerName, layerId, index, displayed) {
  * explicitly does not check the actual data. That's why the score layer needs
  * special handling, so deck.gl is forced to re-render it on parameter changes.
  *
- * @param {LayerMapValue} layerMapValue
- * @param {string} layerName
+ * @param {DisplayedLayerData} layerMapValue
+ * @param {number|string} index index of layer. A number unless this is the
+ * score layer
```

Comment:
I think in the future we might also treat the SVI and SNAP layers like the score layer (index-less) if we go the route of not needing them to be explicitly specified as layer assets. Just a thought.

Reply:
why would they be index-less? They would still have a relative position in the rankings, no? The score is special because it's always last.

Reply:
Yeah, we can auto-populate that in the disaster creation, but I think the map viewer can be agnostic about that?